src/main/java/core/selenium/util/WebDriverMethod.java

```diff
@@ -116,6 +116,19 @@ public Boolean apply(final WebDriver driver) {
         });
     }
 
+    /**
+     * Gets the text in given web element.
```

Comment:
Please, add a whitespace bellow  this line.

Reply:
done